.run/DemoAppWicketJdo.run.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I guess we need to add .run entry with gitignore

Reply:
Actually, this directory holds an IntelliJ run configuration, so is useful for those of us using that IDE.